libs/api/domains/health-directorate/src/lib/health-directorate.service.ts

```diff
@@ -56,12 +56,19 @@ export class HealthDirectorateService {
   }
 
   async updateDonorStatus(auth: Auth, input: DonorStatusInput): Promise<void> {
-    return await this.organDonationApi.updateOrganDonation(auth, {
-      exceptionComment: input.exceptionComment ?? '',
-      isDonor: input.isDonor,
-      exceptions: input.exceptions ?? [],
-      registrationDate: new Date(),
-    })
+    await this.organDonationApi
+      .updateOrganDonation(auth, {
+        exceptionComment: input.exceptionComment ?? '',
+        isDonor: input.isDonor,
+        exceptions: input.exceptions ?? [],
+        registrationDate: new Date(),
+      })
+      .then(() => {
+        return true
+      })
+      .catch(() => {
+        return false
+      })
```

Comment:
this feels weird, what's the purpose of this?

Reply:
Just stupid problem handling because the api definition just returns void, will be updated soon as the swagger is updated 🫠